### ISSUE TITLE
BETTER QUESTING ISSUES AND COMMANDS

### ISSUE THREAD

**Issue by CaiganMythFang** [CaiganMythFang/TheFerretBusinessIssues#5]
There are multiple known issues with Better Questing
## QUEST BOOK SCROLL RESETTING

If you cannot scroll through the quest book, or select a multiple-choice reward, please stop submitting any RF Submission or Large Supply Orders (any quest that uses one of the Submission Stations). There is a bug with the version of BQ in the pack.

This is fixed in the next pack update.
## THE REPUTATION TRACKS IN HQM STAY 0

The HQM reputation tracks are depricated and no longer used. The tracks were left in by mistake, and are already removed for the next update.

Better Questing uses the vanilla Scoreboard system to track reputation instead.
## HOW DO I TRACK MY REPUTATION?

There is no easily accessible quest in BQ to check your reputation. The next update has questlines to track all three reputation tracks, though.

You can check your current reputation with /scoreboard players list playername
## MY SCOREBOARD TRACKS DO NOT UPDATE OR RESET

This is a rare bug I have not been able to track down the cause of.

For now, you can use the commands in the section below to complete specific quests that rely on reputation.
## QUESTS REQUIRE ALL ITEMS IN A TASK

This is a known issue and a fix will be in the next version.
## QUESTS HAVE ODD SYMBOLS AND NUMBERS SOMETIMES

It tends to look like [this](http://i.imgur.com/d8zwdza.png). This is an issue that has been reported to the Better Questing dev [here](https://github.com/Funwayguy/BetterQuesting/issues/68).

This tends to happen in Multiplayer, but has been reported to happen in singleplayer sometimes.

Unfortunately there is no real good workaround for this yet. I will not be adding much more text styles like bold or color until its fixed.

**Comment by CaiganMythFang** [comment]
# BETTER QUESTING : COMMON COMMANDS

## Finding your QuestID

Multiple commands require knowledge of the a quest's Quest ID.

You can find this by holding SHIFT when hovering your mouse over the quest. There will be a number after the quest name, which is the QuestID

## /bq_admin reset
- /bq_admin reset all playername
  - This will reset (playername)'s quest progress. Leave playername blank to reset ALL player's quest progress
- /bq_admin reset QuestID playername
  - This will reset the specific quest for (playername)
    ## /bq_admin complete
- /bq_admin complete QuestID playername
  - This will mark the specific quest completed for (playername)
    ## /bq_admin load
- /bq_admin default load
  - This will load the DefaultQuests file. This is important to use when the pack updates. You can also use the included quest in the BQ Questbook.
    ## /bq_admin edit
- /bq_admin edit
  - This will drop you in and out of edit mode. 

## SCOREBOARD HAS BEEN REMOVED AFTER 0.2.1

**Comment by milana6asmozo** [comment]
I've experienced the same bug twice in two different mod packs and I hope this is the right spot to report.
In both Modern skyblock 3 and Forever stranded LS the quests unlocks themselves without me finishing them and the "new quest unlock" dings goes on repeat and never ceases to stop. 

Since it's only better questing that is affected after the last updates I suspect it is a fairly new bug.
I hope this was useful.
  Milana

**Comment by Gamebuster19901** [comment]
> I've experienced the same bug twice in two different mod packs and I hope this is the right spot to report.
In both Modern skyblock 3 and Forever stranded LS the quests unlocks themselves without me finishing them and the "new quest unlock" dings goes on repeat and never ceases to stop.
>
>Since it's only better questing that is affected after the last updates I suspect it is a fairly new bug.
I hope this was useful.
Milana

You appear to be talking about a bug for Better Questing, not this modpack itself. It appears to have already been reported here: https://github.com/Funwayguy/BetterQuesting/issues/172

**Comment by JoshMing1** [comment]
Hi, I just started playing this modpack yesterday. I was in the "Initial Research" questline, and a lot of the quests I'm turning in are completing, but do not let me claim the rewards and will stay flashing blue saying "unclaimed rewards". I tried doing /bq_admin edit and see if there was a way to help with this there, but there was not. Does anybody know of a possible solution?

**Comment by JoshMing1** [comment]
*Update* It is now every quest that will not let me turn in rewards, not just the few I found halfway through the "Initial Research" questline.

**Comment by CaiganMythFang** [comment]
Is it on any quest that has a Multiple Choice reward selection, and you are selecting one before attempting to claim?

**Comment by JoshMing1** [comment]
So far, they are all on choice reward quests, but the claim button is not highlighted as if I had not completed it in the first place, but it takes the items I turn in and tells me I've completed it. Don't know if whether or not it is a choice quest will affect it.

**Comment by JoshMing1** [comment]
Ok, I'm very dumb. I'm really sorry for wasting your time. All I had to do was click one of the choice rewards to get it to work. One still doesn't work, but that's the one I quest edited. I haven't used Better Questing in this manner before, (where there are so many choice rewards), so I didn't know that's what you had to do. Sorry :(

**Comment by CaiganMythFang** [comment]
It's okay! I was about to ask if you selected one yet, thats when the Claim button lights up. =)

**Comment by JoshMing1** [comment]
Yeah, my problem solving skills & common sense aren't so great today :P

**Comment by JohnnyOasis** [comment]
Hey, so for some reason, I have a quest for flint hatchet that won't complete, and holding shift over the quest won't show me the quest ID so I cant admin complete it. Any thoughts?

**Comment by Funwayguy** [comment]
Why are you commenting in this unrelated issue thread? Please make a new one and describe it in more detail about its setup.

**Comment by JohnnyOasis** [comment]
The thread is better questing issues....this is an issue Im having. Nevermind though. I've decided to go with a different modpack all together

**Comment by CaiganMythFang** [comment]
I'm unsure where I have a quest for a Flint Hatchet, unfortunately. Which quest name/page is it on?

**Comment by Funwayguy** [comment]
oh sorry, did realise it wasn't in the BetterQuesting repo.

Tooltips were moved to Minecraft's built in advanced tooltips which can be enabled with F3+H

**Comment by Morrigon** [comment]
Hi.  I have been trying to get this same issue resolved, the flint axe in better questing to show.  It is causing some issue here as I am attempting to stream the game play and without being able to "detect" the flint axe in the first bit of questing after having crafted the item and having it in my inventory and even having it on my hot bar it still does not reflect as detected.  Is there a fix for this quest, as I remember it very clearly working the first and second attempts I made pre-stream to complete it.  If it would help I will gladly submit in game footage of the issue.

**Comment by Koalasssssssss** [comment]
ok so im playing divine journey and in the quest book I have completed every quest yet it says I need to complete other quests to unlock the next quests, this happens on almost every modpack I play now and whole sections of the quest book wont unlock. Please help me.

**Comment by CaiganMythFang** [comment]
This is not the github for Divine Journey or the other modpacks, nor for Better Questing itself, so I can't really help you much.

I can only guess they have hidden quests you have not hit yet, are not seeing the quests (drag the quest pages around, scroll wheel to zoom in and out), or there is an issue with the pack or your system.

**Comment by SoupKtchn** [comment]
Hey so, i just recently downloaded Better Minecraft but i didn’t get the quest book. Does anyone know why?
